src/service/category_apis.js

```diff
@@ -7,10 +7,10 @@ function useCategories() {
   useEffect(() => 
       axios
       .get("/api/categories")
```

Comment:
Instead of calling async call in useEffect, transfer this to function and implement function accordingly. It's not a good practice.